docs/references/release-notes.md

```diff
@@ -1,5 +1,68 @@
 # Release notes
 
+## Version 0.2.1
+
+### Feature changes and enhancements
+
+- Ragna is now available on
+  [conda-forge](https://github.com/conda-forge/ragna-feedstock)! Check out the
+  [installation instructions](../install.md).
+- A new
+  [extensive tutorial on how to use custom components](../generated/tutorials/gallery_custom_components.md)
```

Comment:
```suggestion
  [extensive tutorial on how to use custom components](../tutorials/gallery_custom_components.py)
```
I'm not sure if I might be missing something here, but this url doesn't seem to work for me